bin/do_username

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env do_username
+
+require 'do_username'
+if ARGV.length < 1
+    puts DOUsername.generate()
+    exit
+end
+
+puts DOUsername.generate(ARGV[0].to_i)
```

Comment:
After #10 adding empty lines in files will be required ;)

Reply:
Oops! Usually I do that lol